subprojects/maven/src/integTest/groovy/org/gradle/api/publish/maven/MavenPublishVersionRangeIntegTest.groovy

```diff
@@ -16,15 +16,17 @@
 
 package org.gradle.api.publish.maven
 
-import org.gradle.integtests.fixtures.ToBeFixedForConfigurationCache
 import org.gradle.integtests.fixtures.publish.maven.AbstractMavenPublishIntegTest
 
 class MavenPublishVersionRangeIntegTest extends AbstractMavenPublishIntegTest {
     def mavenModule = javaLibrary(mavenRepo.module("org.gradle.test", "publishTest", "1.9"))
 
-    @ToBeFixedForConfigurationCache
     void "version range is mapped to maven syntax in published pom file"() {
         given:
+        ('A'..'E').each {
+            javaLibrary(mavenRepo.module("group", "project${it}", "1.0")).withModuleMetadata().publish()
```

Comment:
💄 I still think this code is demanding too much from the reader. You only get to see the word `publish` after scrolling to the right and then you understand what's all about.

Reply:
Reformatted